environment.yml

```diff
@@ -4,19 +4,24 @@ channels:
   - defaults
 dependencies:
   - codecov
+  - configparser
   - coverage
   - flake8-black
   - flake8-docstrings
   - flake8-import-order
+  - mrcfile
   - numba
   - numpy
   - pip
   - pyfftw
   - pytest
   - pytest-cov
+  - pytorch=1.9.0
   - pip:
     - black==20.8b0
     - geomstats
     - git+https://github.com/compSPI/ioSPI.git
     - pre-commit
-    - raster_geometry
\ No newline at end of file
+    - pytorch3d==0.5.0
```

Comment:
there seems to be some issue with `pytorch3d` in the linting/testing

Reply:
we discussed it this morning, relaxing the version requirement might help?